### DISCUSSION TITLE
PDNS Slave Error

### DISCUSSION THREAD

**Discussion by Juju100801** [PowerDNS/pdns#13367] [Q&A]
Hello everyone !
I configured my Slave PDNS server, but I get this message when I want to synchronize the Zones from the Master to the Slave :

`2023-10-15T11:46:18.916556+02:00 ns3 pdns_server[2719]: Unable to find backend willing to host XXXXX.com for potential supermaster 185.45.252.120. Remote nameservers:
2023-10-15T11:46:18.916714+02:00 ns3 pdns_server[2719]: ns3.XXX.com
2023-10-15T11:46:18.916785+02:00 ns3 pdns_server[2719]: ns1.XXX.com
2023-10-15T11:46:18.916841+02:00 ns3 pdns_server[2719]: ns2.XXX.com`

Thank you for your help
Best regards !

**Comment by Habbie** [comment]
Please read https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open and update your post. Please also show us the content of the supermasters table on the intended secondary.
